package.json

```diff
@@ -13,15 +13,19 @@
     "seed:db": "node database/seed.js",
     "reset:db": "mysql -u root < database/schema.sql"
   },
-  "dependencies": {},
+  "dependencies": {
+    "jquery": "^3.4.1"
+  },
   "devDependencies": {
```

Comment:
You don't seem to be using any of these devDependencies (except maybe `morgan`?) as devDependencies. Babel, webpack, express, etc need to be listed as dependencies.